public/widgets/paper-badger-widget.js

```diff
@@ -87,22 +87,44 @@ function splitArray(arr, lookGroup, taxonomyClass)
 	return ttt;
 }
 
+function getEndpoint(confIn, count)
```

Comment:
Code wise this works. Style wise I find having both + and join('/') confusing:
        
        
        function getEndpoint(confIn, count) {
        	var endPoint=[];
            endPoint.push("https://badges.mozillascience.org");
        	if (confIn["article-doi"]) endPoint.push("papers/" + confIn["article-doi"]);
        	if (confIn["orcid"]) endPoint.push("users/" + confIn["orcid"]);
        	endPoint.push("/badges");
        	if (count) endPoint.push("/count" + count)
        	return endPoint.join("/");
        }
        
        function getEndpoint(confIn, count) {
        	var endPoint = "https://badges.mozillascience.org";
        	if (confIn["article-doi"]) endPoint += "papers/" + confIn["article-doi"];
        	if (confIn["orcid"]) endPoint += "users/" + confIn["orcid"];
        	endPoint += "/badges";
        	if (count) endPoint += "/count" + count;
        	return endPoint;
        }
        
Would either of those be clearer? I'm not sure.